{"links": {"self": {"href": "data/repositories/osrf/subt/issues/353/comments/56815873.json"}, "html": {"href": "#!/osrf/subt/issues/353#comment-56815873"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/353.json"}}, "type": "issue", "id": 353, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "RGBD Camera error upon launch"}, "content": {"raw": "Issue #374 was marked as a duplicate of this issue.", "markup": "markdown", "html": "<p>Issue <a href=\"#!/osrf/subt/issues/374/cv_bridge-exception\" rel=\"nofollow\" title=\"cv_bridge exception\" class=\"ap-connect-link\"><s>#374</s></a> was marked as a duplicate of this issue.</p>", "type": "rendered"}, "created_on": "2020-04-06T18:27:36.690398+00:00", "user": {"display_name": "Arthur Schang", "uuid": "{1403972a-9b64-442b-a69c-d7e9557e1d87}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B1403972a-9b64-442b-a69c-d7e9557e1d87%7D"}, "html": {"href": "https://bitbucket.org/%7B1403972a-9b64-442b-a69c-d7e9557e1d87%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/initials/AS-0.png"}}, "nickname": "Arthur Schang", "type": "user", "account_id": "557058:14b7fd50-d65a-47a8-825c-fa89681d07bd"}, "updated_on": null, "type": "issue_comment", "id": 56815873}